client/my-sites/plans-v2/plans-column/index.tsx

```diff
@@ -102,6 +102,25 @@ const PlansColumn = ( { duration, onPlanClick, productType, siteId }: PlanColumn
 				...product,
 				owned: product.productSlug === currentPlan,
 				legacy: false,
+			} ) )
+			// If product offers 'realtime' and 'daily' options, AND product is not already owned,
+			// append "Available Options: Real-Time and Daily" to the product description.
+			.map( ( product ) => ( {
+				...product,
+				description:
+					product.subtypes?.filter( ( subtype ) => /_(daily|realtime)$/.test( subtype ) ).length >=
+						2 && product.productSlug !== currentPlan ? (
```

Comment:
In the case of Jetpack Security, you could purchase Security _Daily_ or Security _Real-time_, but never Jetpack Security. I believe the condition should be updated to reflect that. Something like `! product.subtypes.includes(currentPlan)`.